docs/docs/recce-cloud/share-recce-session-securely.md

```diff
@@ -11,30 +11,43 @@ Recce Cloud allows you to share your Recce validation results with full context,
 
 !!! Note
 
-    Please note that Recce users with the link can visit your sharing session. If you need to restrict access, please contact us.
+    Please note that anyone with the link can visit your shared Recce after sign in Recce Cloud. If you need to restrict access, [please contact us](https://cal.com/team/recce/chat).
 
 ## How to use
 
-1. Obtain API token from Recce Cloud
-    
-    To connect to your Recce Cloud account, you need an API token. Please copy your API Token from the [personal settings page](https://cloud.datarecce.io/settings#tokens) in Recce Cloud.
+When you're ready to share your lineage exploration, query results, or validation checklist, simply click **Share** in Recce.
+
+The first time you do this, you'll need to connect your local Recce to Recce Cloud. This requires signing in and setting up an API token. Once connected, Recce Cloud will host your state file securely, allowing you to share a link that others can open in their browsers.
+
+1. Enable Recce Sharing
+
+    To start sharing, launch Recce server and click the **Enable Sharing** button.
+
+    ![Recce Server](../../assets/images/recce-cloud/recce-server-enable-sharing-fs8.png){: .shadow}
+
+1. Sign in to Recce Cloud and get your API token
+
+    Copy your API token from the [personal settings page](https://cloud.datarecce.io/settings#tokens) in Recce Cloud.
+
     ![Recce API Token](../../assets/images/recce-cloud/setting-page-api-token-fs8.png){: .shadow}
 
 1. Add the token to `.recce/profile.yml`
 
-    For convenience, you can enter your API token in `.recce/profile.yml`, located by default in your home directory. 
+    For convenience, you can add your API token in `.recce/profile.yml`, located by default in your home directory. 
     ```yaml
     user_id: <your_user_id>
     api_token: <your_api_token>
     ```
-    Or, you can use `--api-token` flag with commend for one-time use. e.g.,
+    Alternatively, for one-time use, you can use `--api-token` flag with commend. e.g.,
     ```bash
     recce server --api-token <your_api_token>
     ```
 
-1. Start Recce server
+1. Relaunch Recce server
+
+    After adding the API token, restarting Recce server is required to load the new configuration.
+    Once it's running, you'll see the **Share** button, then you can click it to get the link on top.
 
-    After launching, you'll see the __Share__ button at the top, then you can click it to get the link.
     ![Recce Share From Server](../../assets/images/recce-cloud/recce-share-from-server-fs8.png){: .shadow}
```

Comment:
can you replace by a screenshot that with the URL? show the result after click share.